Snippets/Snippets_Misc/EF EDMX/MySample.edmx.diagram

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
to allow VS to generate a DB context, yes, as before we can do it manually but why?